mne/epochs.py

```diff
@@ -1220,17 +1224,26 @@ def _get_data(self, out=True, verbose=None):
             self._getitem(good_idx, None, copy=False, drop_event_id=False,
                           select_data=False)
 
-        return data if out else None
+        if out:
+            if picks is None:
+                return data
+            else:
+                picks = _picks_to_idx(self.info, picks)
+                return data[:, picks]
+        else:
+            return None
+
 
-    def get_data(self):
+    def get_data(self, picks=None):
         """Get all epochs as a 3D array.
 
         Returns
         -------
         data : array of shape (n_epochs, n_channels, n_times)
             A view on epochs data.
+        %(picks_all)s
```

Comment:
@larsoner is this how the docstring should be?

Reply:
It either needs `@verbose` decorator (and arg, so not relevant here) or `@fill_doc` decorator, otherwise it looks good